Sources/AloeStackView/AloeStackViewController.swift

```diff
@@ -34,9 +34,20 @@ open class AloeStackViewController: UIViewController {
     view = stackView
   }
 
+  open override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+    if automaticallyFlashScrollIndicators {
+        stackView.flashScrollIndicators()
+    }
+  }
+
   // MARK: Public
 
   /// The stack view this controller manages.
   public let stackView = AloeStackView()
 
+  /// Automatically displays the scroll indicators in the stack view momentarily.
```

Comment:
Clarity: “When `true`, automatically displays the scroll indicators in the stack view momentarily whenever the view appears.”